app/models/agents/telegram_agent.rb

```diff
@@ -79,7 +79,11 @@ def receive_event(event)
     def send_telegram_message(method, params)
       params[:chat_id] = interpolated['chat_id']
       params[:parse_mode] = interpolated['parse_mode'] if interpolated['parse_mode'].present?
-      HTTMultiParty.post telegram_bot_uri(method), query: params
+      txt_message = params[:text].scan /.{1,4096}\W/m
```

Comment:
Not to break in the middle of a word, I guess, but I suspect the last word would be lost if the text ends with a letter (`\w`).